docs/source/markdown/options/network.md

```diff
@@ -74,3 +74,6 @@ Valid _mode_ values are:
     - **pasta:-T,5201**: enable forwarding of TCP port 5201 from container to
         host, using the loopback interface instead of the tap interface for improved
         performance
+
```

Comment:
If you want the note below to appear, in the man page, together with the paragraph above (same indentation), you need to drop this newline and add a trailing `\` above -- that's (unfortunately) how this Markdown is converted.

If that was not the intention, you can leave it like you did. I checked both versions and I really can't decide myself which one looks better -- I think it's fine either way.

Reply:
It should be under the pasta point not the examples list, if I render this locally it is on the same level as `Some examples:` which is what I want

Reply:
This is how it renders for me, `man ./docs/build/man/podman-run.1` with 80 columns, man 2.11.0 and GNU nroff 1.22.4:

```
                Some examples:

                • pasta:--map-gw: Allow the container to  directly  reach  the
                  host using the gateway address.

                • pasta:--mtu,1500:  Specify  a 1500 bytes MTU for the tap in‐
                  terface in the container.

                • pasta:--ipv4-only,-a,10.0.2.0,-n,24,-g,10.0.2.2,--dns-for‐
                  ward,10.0.2.3,-m,1500,--no-ndp,--no-dhcpv6,--no-dhcp, equiv‐
                  alent to default slirp4netns(1) options: disable  IPv6,  as‐
                  sign  10.0.2.0/24  to  the  tap0 interface in the container,
                  with gateway 10.0.2.3, enable  DNS  forwarder  reachable  at
                  10.0.2.3,  set  MTU  to  1500 bytes, disable NDP, DHCPv6 and
                  DHCP support.

                • pasta:-I,tap0,--ipv4-only,-a,10.0.2.0,-n,24,-g,10.0.2.2,--dns-
                  forward,10.0.2.3,--no-ndp,--no-dhcpv6,--no-dhcp,  equivalent
                  to default slirp4netns(1)  options  with  Podman  overrides:
                  same as above, but leave the MTU to 65520 bytes

                • pasta:-t,auto,-u,auto,-T,auto,-U,auto: enable automatic port
                  forwarding based on observed bound ports from both host  and
                  container sides

                • pasta:-T,5201:  enable forwarding of TCP port 5201 from con‐
                  tainer to host, using the loopback interface instead of  the
                  tap interface for improved performance

              NOTE:  For backwards compatibility reasons if you have a network
              named pasta podman will use this instead.  In order to  use  the
              pasta binary you need to remove this network.
```

I can look into how this is built on my system if it helps -- `./test/tools/build/go-md2man` doesn't give me a version string but I suppose it just matches git `HEAD`.

Reply:
you are right, I only looked the the rendered markdown not the rendered man page.

Reply:
By the way, that's because of:

```
Some examples:
.RS
.IP \(bu 2

[...]

.RE
NOTE: For backwards compatibility reasons if you have a network named \fB\fCpasta\fR podman will use this instead.
In order to use the pasta binary you need to remove this network.

.RE
```

that first `.RE` (margin reset), which shouldn't be there. I _think_ it's because `md2man` mistakes the whole thing as a `BlockQuote` instead of a `Paragraph` in `RenderNode()`.

Reply:
On the other hand, if you want to keep it as a [paragraph](https://spec.commonmark.org/0.30/#paragraphs) as defined by CommonMark, you would have to add whitespace on that blank line, because "Paragraphs can contain multiple lines, but no blank lines", and yet you want line breaks. And that's rejected by some linter that will scan this file (I don't remember the details).

I can't think of any solution with the given tools at the moment -- I'd rather leave it like you have it.

Reply:
agreed, lets keep it as it.